spec/stoplight/notifier/generic_spec.rb

```diff
@@ -47,4 +47,24 @@
   it 'is a module' do
     expect(described_class).to be_a(Module)
   end
+
+  describe '#notify' do
+    let(:light) { Stoplight::Light.new(name) {} }
+    let(:name) { ('a'..'z').to_a.shuffle.join }
+    let(:from_color) { Stoplight::Color::GREEN }
+    let(:to_color) { Stoplight::Color::RED }
+    let(:notifier) { notifier_class.new(double.as_null_object) }
+    let(:error) { nil }
+    let(:notifier_class) do
+      Class.new do
+        include Stoplight::Notifier::Generic
+      end
+    end
+
+    it 'has to implement the #put method' do
```

Comment:
We can also test that we call the `formatter#call` method with correct arguments and return a formatted message from the `#notify` method, but it's totally optional.

Reply:
Agreed